src/api/app/controllers/webui/groups/bs_requests_controller.rb

```diff
@@ -2,6 +2,12 @@ module Webui
   module Groups
     class BsRequestsController < WebuiController
       before_action :set_group
+      before_action :require_login
+      before_action :redirect_legacy
+      before_action :set_bs_request
+
+      include Webui::RequestsFilter
+      include BsRequest::FindFor::UserGroupMixin
```

Comment:
Why?

Reply:
This is a left over from my last refactoring, when I was trying to stop using the `FindFor::Query`. I'll remove it.